### PR TITLE
Send Registration XML to Jembi

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review, but land #90 first.

File: registration/tasks.py [hunk 1]
```diff
@@ -76,6 +79,228 @@ def build_jembi_json(registration):
...
+        value_element.set('displayName', preg_display_name)
+
+        entry_relationship_element = \
+            observation_element.find('entryRelationship')
+        for value_element in entry_relationship_element.iter('value'):
+            value_element.set('value', due_date)
+
+    root.set("xmlns", "urn:hl7-org:v3")
+
+    # tree.write('Mikeoutputlxml.xml', encoding='UTF-8', xml_declaration=True)
```
Comment (imsickofmaps) [inline registration/tasks.py]: remove comment

File: registration/tasks.py [hunk 2]
```diff
@@ -259,6 +484,42 @@ def jembi_post_json(registration_id):
...
+        headers = {
+            'Accept-Encoding': 'gzip',
+            'Content-Type': 'multipart/form-data; boundary=yolo'
+        }
+        auth = (settings.JEMBI_USERNAME, settings.JEMBI_PASSWORD)
+
+        try:
+            result = requests.post(api_url, headers=headers, data=data,
+                                   auth=auth, verify=False)
+            return result.text
```
Comment (imsickofmaps) [inline registration/tasks.py:508]: We need to capture the response and if it's successful mark the reg in the model at sent so we can requeue any issues

File: registration/tests.py [hunk 3]
```diff
@@ -481,6 +481,10 @@ def test_create_registration_fires_tasks(self):
                       "http://test/v2/json/subscription",
                       body='jembi_post_json task', status=201,
                       content_type='application/json')
+        responses.add(responses.POST,
+                      "http://test/v2/registration/net.ihe/DocumentDossier",
+                      body='jembi_post_xml task', status=201,
```
Comment (imsickofmaps) [inline registration/tests.py]: Proper response here `Request added to queue` is good, everything else is bad. Also it's 202's from Jembi.

**Comment by imsickofmaps** [comment]
First review done

File: registration/tasks.py (hunk 2 above)
Comment (gsvr) [inline registration/tasks.py:508]: Issue opened: https://github.com/praekelt/ndoh-control/issues/99

**Pull request merged**